### PR TITLE
byobu: 5.133 -> 6.12

### LINKED ISSUE #290168: Byobu has permanently moved development to GitHub
- Package name: Byobu
- Latest released version: 6.12

- Current version on the unstable channel: 5.133
- Current version on the stable/release channel: 5.133

Byobu development has permanently moved from Launchpad to GitHub
[https://bazaar.launchpad.net/~kirkland/byobu/trunk/revision/2646](url)
[https://github.com/dustinkirkland/byobu](url)

Note for maintainers: Please tag this issue in your PR.

---

Add a :+1: [reaction] to [issues you find important].

[reaction]: https://github.blog/2016-03-10-add-reactions-to-pull-requests-issues-and-comments/
[issues you find important]: https://github.com/NixOS/nixpkgs/issues?q=is%3Aissue+is%3Aopen+sort%3Areactions-%2B1-desc

### PR DESCRIPTION
**Pull request by AndersonTorres**
## Description of changes

Closes https://github.com/NixOS/nixpkgs/issues/290168 and https://github.com/NixOS/nixpkgs/pull/290187

## Things done

- Built on platform(s)
  - [ ] x86_64-linux
  - [ ] aarch64-linux
  - [ ] x86_64-darwin
  - [ ] aarch64-darwin
- For non-Linux: Is sandboxing enabled in `nix.conf`? (See [Nix manual](https://nixos.org/manual/nix/stable/command-ref/conf-file.html))
  - [ ] `sandbox = relaxed`
  - [ ] `sandbox = true`
- [ ] Tested, as applicable:
  - [NixOS test(s)](https://nixos.org/manual/nixos/unstable/index.html#sec-nixos-tests) (look inside [nixos/tests](https://github.com/NixOS/nixpkgs/blob/master/nixos/tests))
  - and/or [package tests](https://nixos.org/manual/nixpkgs/unstable/#sec-package-tests)
  - or, for functions and "core" functionality, tests in [lib/tests](https://github.com/NixOS/nixpkgs/blob/master/lib/tests) or [pkgs/test](https://github.com/NixOS/nixpkgs/blob/master/pkgs/test)
  - made sure NixOS tests are [linked](https://nixos.org/manual/nixpkgs/unstable/#ssec-nixos-tests-linking) to the relevant packages
- [ ] Tested compilation of all packages that depend on this change using `nix-shell -p nixpkgs-review --run "nixpkgs-review rev HEAD"`. Note: all changes have to be committed, also see [nixpkgs-review usage](https://github.com/Mic92/nixpkgs-review#usage)
- [ ] Tested basic functionality of all binary files (usually in `./result/bin/`)
- [24.05 Release Notes](https://github.com/NixOS/nixpkgs/blob/master/nixos/doc/manual/release-notes/rl-2405.section.md) (or backporting [23.05](https://github.com/NixOS/nixpkgs/blob/master/nixos/doc/manual/release-notes/rl-2305.section.md) and [23.11](https://github.com/NixOS/nixpkgs/blob/master/nixos/doc/manual/release-notes/rl-2311.section.md) Release notes)
  - [ ] (Package updates) Added a release notes entry if the change is major or breaking
  - [ ] (Module updates) Added a release notes entry if the change is significant
  - [ ] (Module addition) Added a release notes entry if adding a new NixOS module
- [ ] Fits [CONTRIBUTING.md](https://github.com/NixOS/nixpkgs/blob/master/CONTRIBUTING.md).

---

Add a :+1: [reaction] to [pull requests you find important].

[reaction]: https://github.blog/2016-03-10-add-reactions-to-pull-requests-issues-and-comments/
[pull requests you find important]: https://github.com/NixOS/nixpkgs/pulls?q=is%3Aopen+sort%3Areactions-%2B1-desc

### CODE REVIEW

**Comment by AndersonTorres** [comment]
Remember to remove qknight from maintainers

File: pkgs/by-name/by/byobu/package.nix [hunk 1]
```diff
@@ -0,0 +1,100 @@
...
+    cp --remove-destination $out/bin/byobu $out/bin/byobu-screen
+    cp --remove-destination $out/bin/byobu $out/bin/byobu-tmux
+
+    for file in $out/bin/byobu*; do
+      # We don't use the usual "-wrapped" suffix because arg0 within the shebang
+      # scripts points to the filename and byobu matches against this to know
+      # which backend to start with
+      bname="$(basename $file)"
+      mv "$file" "$out/bin/.$bname"
+      makeWrapper "$out/bin/.$bname" "$out/bin/$bname" --argv0 $bname \
```
Comment (SuperSandro2000) [inline pkgs/by-name/by/byobu/package.nix:74]:
```suggestion
      makeWrapper "$out/bin/.$bname" "$out/bin/$bname" \
        --argv0 $bname \
```

File: pkgs/by-name/by/byobu/package.nix [hunk 2]
```diff
@@ -0,0 +1,100 @@
...
+        --set BYOBU_PATH ${lib.escapeShellArg (lib.makeBinPath [ vim bc ])} \
```
Comment (SuperSandro2000) [inline pkgs/by-name/by/byobu/package.nix:75]:
```suggestion
        --set BYOBU_PATH ${lib.makeBinPath [ vim bc ]} \
```
we don't need to escape that

**Review by SuperSandro2000 — APPROVED** [review]

**Review by Samasaur1 — APPROVED** [review]
Builds and runs on aarch64-darwin, x86_64-darwin, and x86_64-linux

**Review by berbiche — APPROVED** [review]
Result of `nixpkgs-review pr 290328` run on x86_64-linux [1](https://github.com/Mic92/nixpkgs-review)
<details>
  <summary>1 package built:</summary>
  <ul>
    <li>byobu</li>
  </ul>
</details>

I didn't test all the functionalities, but the basics seem to work.

**Pull request merged**